core/pkg/service/flag-evaluation/flag_evaluator.go

```diff
@@ -283,10 +285,12 @@ func errFormat(err error) error {
 		return connect.NewError(connect.CodeNotFound, fmt.Errorf("%s, %s", ErrorPrefix, err.Error()))
 	case model.TypeMismatchErrorCode:
 		return connect.NewError(connect.CodeInvalidArgument, fmt.Errorf("%s, %s", ErrorPrefix, err.Error()))
-	case model.DisabledReason:
+	case model.FlagDisabledErrorCode:
```

Comment:
why this change?

Reply:
It was a bug, it is a reason and not an error

Reply:
Nice catch!